osf-cas/templates/configmap.yaml

```diff
@@ -319,6 +319,352 @@ httpd/conf.d/shib.conf: |-
   </VirtualHost>
 {{- end -}}
 {{- define "cas.jetty.inlineconfigs" }}
+config/cas.properties: |-
+  ########################################################################################################################
+  # Server Settings
+  ########################################################################################################################
+  # Server
+  #
+  server.port=${TOMCAT_HTTP_PORT:8080}
+  # Servlet
+  server.servlet.contextPath=/
+  #
+  # SSL
+  server.ssl.enabled=false
+  #
+  # CAS Server
+  cas.server.name=https://{{ .Values.casDomain }}
+  cas.server.prefix=${cas.server.name}
+  # cas.server.scope={{ .Values.casDomain }}
+  #
+  # Tomcat Server
+  #
+  cas.server.tomcat.server-name=OSF CAS
+  # Enable additional HTTP connections for the embedded Tomcat container (when SSL is enabled by default)
+  # cas.server.tomcat.http.port=${TOMCAT_HTTP_PORT:80}
+  # cas.server.tomcat.http.protocol=org.apache.coyote.http11.Http11NioProtocol
+  # cas.server.tomcat.http.enabled=true
+  # cas.server.tomcat.http.attributes=
+  # e.g. cas.server.tomcat.http.attributes.{attribute-name}={attributeValue}
```

Comment:
If we ended up stick with this design where `cas.properties` (and other files) has to be added via charts, let's remove unused / unnecessary properties such as the quoted example above.